### ISSUE TITLE
I don't have the lyrics but others features work

### ISSUE THREAD

**Issue by LeMaxicien** [mrpond/BlockTheSpot#537]
![image](https://github.com/mrpond/BlockTheSpot/assets/165584772/b5fdfa40-a8f7-4901-a9ce-19e8665b05c4)

**Comment by Vilo659** [comment]
i have the same issue pal

**Comment by ampethamine** [comment]
Comments can be unblocked through the debug window. Click on the three dots upper left corner > Develop > Show Debug Window >Remote Configs > Search for lyric > Set "enableLyricsUpsell" to false.

**Comment by ampethamine** [comment]
Also duplicate of #535

**Comment by grloper** [comment]
same, no lyrics

**Comment by invinciblevenom** [comment]
> Some of the lyrics can be unblocked through the debug window but you still need premium to view full lyrics. Click on the three dots upper left corner > Develop > Show Debug Window >Remote Configs > Search for lyric > Set "enableLyricsUpsell" to false.

This does remove the premium banner but unlocks lyrics partially and they aren't time synced.

**Comment by invinciblevenom** [comment]
> same, no lyrics

They are not working after 1.2.37.701.ge66eb7bc update.

**Comment by ChiefEKD19** [comment]
fully uninstalled spotify and downgraded to a version that was working previously (1.2.36.959) and i still cant see lyrics after using the patch?

**Comment by ampethamine** [comment]
> fully uninstalled spotify and downgraded to a version that was working previously (1.2.36.959) and i still cant see lyrics after using the patch?

Lyrics are now a server sided premium feature and won't work on any version now

**Comment by invinciblevenom** [comment]
> > fully uninstalled spotify and downgraded to a version that was working previously (1.2.36.959) and i still cant see lyrics after using the patch?
> 
> Lyrics are now a server sided premium feature and won't work on any version now

This applies to Spotify Desktop?

**Comment by Nuzair46** [comment]
duplicate #535
